pages/clubs.jsx

```diff
@@ -439,7 +439,7 @@ var ModalAddOrChangeYourClub = React.createClass({
                onChange={this.handleLocationChange} />
             </fieldset>
             <fieldset>
-              <label htmlFor={idPrefix + "website"}>What is your Club&lsquo;s website?</label>
+              <label htmlFor={idPrefix + "website"}>What is your Club&lsquo;s website?<span className="optional-text">optional</span></label>
```

Comment:
Oh, it's because the autocomplete widget we're using doesn't support that attribute :/ It's checked by our custom fallback validation code, though, so it is indeed required!